src/clojure/nrepl/server.clj

```diff
@@ -210,7 +210,9 @@
                  :else
                  (inet-socket bind port))
         server (Server. ss
+                        (when-not socket bind)
```

Comment:
I'm wondering if we shouldn't raise an error if both host and socket are provided. I'm also thinking that at some point we should add some `:host` key as well (an alias for `:bind`), because even I forget that `:bind` is the host. 😄

Reply:
@bbatsov Do you mean we should add a `:bind` key? The *local* here is `bind`, but the field on the record I've added is `:host` (since that seems to be what other parts of the program look for).

>  raise an error if both host and socket are provided

I think that would make sense. Do you think that would be better added down here in the server code, or as an input validation in the `cmdline` namespace?

Reply:
Probably here would be best. 

> @bbatsov Do you mean we should add a :bind key? The local here is bind, but the field on the record I've added is :host (since that seems to be what other parts of the program look for).

Yeah, I know - I was just wondering if we shouldn't use the `host` terminology more pervasively. I think the in most places of the code we use `bind`, because something in the java.net code was using this terminology. Anyways, let's leave this as is and not expand the scope.